src/components/markdown/markdown.test.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import Enzyme, { mount } from 'enzyme';
+import { MarkdownSlide, MarkdownSlideSet } from './markdown';
+import Adapter from 'enzyme-adapter-react-16';
+import Deck from '../deck/deck';
+import { ListItem } from '../typography';
+import Appear from '../appear';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+const mountInsideDeck = tree => {
+  return mount(<Deck>{tree}</Deck>);
+};
+
+describe('<MarkdownSlide />', () => {
+  it('should generate standard unordered lists by default', () => {
+    const wrapper = mountInsideDeck(
+      <MarkdownSlide>{`
+        - One
+        - Two
+        - Three
+      `}</MarkdownSlide>
+    );
+
+    expect(wrapper.find('ul')).toHaveLength(1);
+    expect(wrapper.find(ListItem)).toHaveLength(3);
+    expect(wrapper.find(Appear)).toHaveLength(0);
+  });
+
+  it('should generate animated list items with animateListItems', () => {
+    const wrapper = mountInsideDeck(
+      <MarkdownSlide animateListItems>{`
+        - One
+        - Two
+        - Three
+      `}</MarkdownSlide>
+    );
+
+    expect(wrapper.find('ul')).toHaveLength(1);
+    expect(wrapper.find(Appear)).toHaveLength(3);
```

Comment:
I'd love to also test for the existence of `ListItem` components, but those only appear as you step through the slide. I wasn't sure if figuring out how to imperatively do that was worth the time. The enzyme tests are already in the realm of "testing implementation details", so this set of assertions felt sufficient.

In the future, we might consider moving to `react-testing-library` for testing to move away from testing implementation details. My ideal test would be along the lines of:

- Mount the slide.
- Check that none of the list items are present.
- Simulate a click.
- Check that the first list item is present (searching by text), but not the last two.
- Simulate two more clicks.
- Check that all list items are present (searching by text).